Tutorials/Creating-Basic-Site/Articles-Parent-and-Article-Items/index.md

```diff
@@ -2,109 +2,132 @@
+8. Select **Articles Item** and click **Save**.
+9. Go to the **List View** tab.
+10. Toggle **Enable List view button** and click **Save**.
+
+To create **Articles Item** document type, follow these steps:
+
+1. Go to **Settings**.
+2. In the **Settings** pane, click the **...** next to the **Document Types** in the tree.
+3. Click **Document Type with Template**.
+    ![Document type with template](images/Document_type_with_template.png)
+4. Enter a **Name** for the **Document Type**. Let's call it _Articles Item_.
+5. Let's add two fields with the following specifications:
+    | Group   | Field Name      | Alias          | Data Type        |
+    |---------|-----------------|----------------|------------------|
+    | Content | Article Title   | articleTitle   | TextBox          |
+    | Content | Article Content | articleContent | Rich Text Editor |
+
+    ![Article Item Document Type Data Properties](images/figure-39-articles-item-v8.png){: data-fancybox}
+6.  Navigate to the **Home Page** node and go to the **Permissions** tab.
+7. In the **Allowed child node types**, click **Add child**. The **Choose child node** window opens.
+8. Select **Articles Main** and click **Save**.
+
+## Creating the Content Node
+
+To add a content node:
+
+1. Go to **Content**.
+2. Click the **...** next to the **HomePage** and select **Articles Main**.
+3. Enter the name for the article. We are going to call it _Articles_.
+4. Enter the **Article Title**, **Article Content**, and click **Save**.
+   When you click on Save, you will notice a list view is created. The child nodes are placed in a list making it easier to view them. You can create new nodes from this section.
+   
+   :::tip
+    If you do not see the list view, try refreshing the page.
+   :::
+5. Let's add two child nodes called **Article 1**, **Article 2**, and click **Save**.
+    ![Content Tree With Articles](images/figure-40-articles-created-v8.png)
+
+## Updating the Template
+
+To update the **Articles Main** template, follow these steps:
+
+1. Go to **Settings**.
+2. In the **Templating** section, expand the **Templates** folder. You should see a template titled **_Articles Main_**.
+3. In the **Master template** select **Master** and click **Save**.
+4. Copy the template content from the **_Simple Content Page_**  template and paste the content into **Articles Main**.
+    
+    :::warn
+    Take care when pasting the template not to overwrite the first line `@inherits Umbraco.Web.Mvc.UmbracoViewPage<ContentModels.ArticlesMain>`. If you get an error when loading the page ensure the last part in <> brackets matches your Document Type alias.
+    :::
+5. Replace the static text within the `<!-- Jumbotron, w title -->` tags with the Model.Value references. For example:  Add references to **_articlesTitle_** and **_articlesBodyText_**.
+ ![Articles Main Template](images/articles-main-template.png)
```

Comment:
The code in the screenshot here, is not the same as I get, once I copy over the code from the Simple Content Page..

I have the following copied:
```
<!-- Jumbotron, w title -->
    <div class="jumbotron text-center jumbotron-fluid">
            <div class="container">
                <h1 class="display-1">@Model.Value("pageTitle")</h1>
            </div>
        </div>

<!-- Main -->
<section id="main" class="wrapper">
    <div class="container">

        <p>@Model.Value("bodyText")</p>
    </div>
</section>  
```

I think either the image/snippet here need to be updated, or it needs to be updated in the article where the "Simple Content Page" is set up!